### ISSUE TITLE
Update app without App Store review

### ISSUE THREAD

**Issue by riderx** [capacitor-community/proposals#153]
## Plugin Request
Plugin to update your app without App Store review 

**Name**: capacitor-updater
**Package**: `@capacitor-community/capacitor-updater`
https://github.com/riderx/capacitor-updater

The plugin work and it's use in product in my app.

the scope of the plugin is super slim and you need to manage everything yourself.

i was asked to make a paid version with a auto update system, I'm working on it for January.

__Update__ : paid version it __OUT__ [Capgo](https://capgo.app) automatic update server starting at $14

### Platform(s)
- ios
- android

### Existing Solutions
Paid solution made by ionic, 500 dollar by month minimum

**Comment by alexcroox** [comment]
This is a duplicate of https://github.com/capacitor-community/proposals/issues/43

**Comment by riderx** [comment]
@alexcroox agree, I was wandering if I create issue maybe the discussion can go around this plugin and see if it’s accepted or not in community repo

**Comment by muuvmuuv** [comment]
Isn't this already provided by [AppFlow](https://ionic.io/appflow) by Ionic?

**Comment by riderx** [comment]
@muuvmuuv yes, if you need the whole Appflow, this is wonderful.
if you just want live updates, it doesn't make it a good deal

**Comment by gergomiklos** [comment]
For anyone finding this from search: you don't need a proposal, this already works. OTA / live-update plugins push your web layer (JS/HTML/CSS) to installed apps without a store resubmit. It's compliant as long as you don't change the app's core purpose — Apple's guideline 2.5.2 explicitly allows code run by WebKit/JavaScriptCore, and Google Play's rule is about native executable code, not webview JS.

Capacitor options: Capgo, Capawesome, OtaKit (I maintain OtaKit — MIT, https://github.com/OtaKit/otakit). The one native constraint: anything touching plugins, permissions, or native code still needs a store build. Everything else goes over the air.

Where the compliance line actually sits, with the guideline quoted: https://otakit.app/blog/does-apple-allow-live-updates
